### ISSUE TITLE
F17 has no corresponding WCAG failure ?

### ISSUE THREAD

**Issue by DavidMacDonald** [wet-boew/wet-boew-wpss#79]
1.3.1, 4.1.1 F17: Failure of Success Criterion 1.3.1 and 4.1.1 due to insufficient information in DOM to determine one-to-one relationships (e.g. between labels with same id) in HTML

I don't see this in WCAG https://www.w3.org/TR/WCAG20-TECHS/F17

**Comment by irwink** [comment]
The tool has been updated to report duplicate  ID errors under F77: Failure of Success Criterion 4.1.1 due to duplicate values of type ID (https://www.w3.org/TR/WCAG20-TECHS/F77).

**Comment by irwink** [comment]
Change included in version 6.1.0.
